site/docs/advanced/transformers.md

```diff
@@ -92,7 +92,7 @@ They solve this problem:
 
 ```ts
 import { Api, Bot, Context } from "grammy";
-import { SomeApiFlavor, someContextFlavor, somePlugin } from "some-plugin";
+import { SomeApiFlavor, SomeContextFlavor, somePlugin } from "some-plugin";
```

Comment:
Not synced with other languages.

Reply:
That's weird. We did that changes in commit https://github.com/grammyjs/website/pull/434/commits/8da525f33261865e0893ec168b49bdd5d039fb8e. And they are still persisted in the latest commit https://github.com/grammyjs/website/pull/434/commits/a00ed22ca34fbb96e31843e82f38664246bd5c5d. Should we just ignore these suggestions or keep committing them?